packages/web3-eth/test/integration/web3_eth/send_transaction.test.ts

```diff
@@ -0,0 +1,121 @@
+			const transaction: Transaction = {
+				from: accounts[0],
+				data: greeterContractDeploymentData,
+				gas: '0x740b8',
+			};
+			const response = await web3Eth.sendTransaction(transaction);
+			expect(response.status).toBe('0x1');
+			expect(response.contractAddress).toBeDefined();
+
+			greeterContractAddress = response.contractAddress as string;
+		});
+
+		it('should update greet in contract', async () => {
+			const contractFunctionCall =
+				'0xa4136862000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000023432000000000000000000000000000000000000000000000000000000000000';
+			const transaction: Transaction = {
+				from: accounts[0],
+				to: greeterContractAddress,
+				data: contractFunctionCall,
+			};
+			const response = await web3Eth.sendTransaction(transaction);
+			expect(response.status).toBe('0x1');
+		});
+	});
+
+	describe('Transaction Types', () => {
+		it('should send a successful type 0x0 transaction', async () => {
+			const transaction: Transaction = {
+				from: accounts[0],
+				to: '0x0000000000000000000000000000000000000000',
+				value: '0x1',
+				type: '0x0',
+			};
+			const response = await web3Eth.sendTransaction(transaction);
+			expect(response.type).toBe('0x0');
+			expect(response.status).toBe('0x1');
```

Comment:
Along with tx status, I think we should also check that provided from, to, value and type match with mined tx via `eth_getTransactionReceipt`, for testing any of these are not modified by lib or dependent libs.

Reply:
same suggested items should be checked for following tests of 0x1 and 0x2 txs

Reply:
Addressed by [this](https://github.com/ChainSafe/web3.js/pull/5056/commits/d77232fb8c24003dd8d03830ab00bcc97f510495) commit